test_data/pil/add.pil

```diff
@@ -0,0 +1,5 @@
+namespace Add(8);
+    let a;
```

Comment:
You could add hints to `a` and `b`, to make it a bit more interesting! Now I think witgen just sets everything to 0 because it happens to satisfy the constraints.

Reply:
Yeah I think 0 is fine